docs/python/environments.md

```diff
@@ -4,14 +4,16 @@ Area: python
 TOCTitle: Environments
 ContentId: 8fe4ca8b-fc70-4216-86c7-2c11b6c14cc6
 PageTitle: Using Python Environments in Visual Studio Code
-DateApproved: 04/08/2022
+DateApproved: 04/20/2022
 MetaDescription: Configuring Python Environments in Visual Studio Code
 MetaSocialImage: images/tutorial/social.png
 ---
 # Using Python environments in VS Code
 
 This article discusses the helpful Python environments features available in Visual Studio Code. An "environment" in Python is the context in which a Python program runs and consists of an interpreter and any number of installed packages.
 
+> **Note**: If you'd like to become more familiar with the Python programming language, review [More Python resources](./environments.md/#more-python-resources).
```

Comment:
The link can just be #more-python-resources. You don't to prefix with the filename